modules/tinymce/src/themes/silver/test/ts/browser/throbber/ThrobberPopupTest.ts

```diff
@@ -43,6 +52,64 @@ describe('browser.tinymce.themes.silver.throbber.ThrobberPopupTest', () => {
     });
   });
 
+  it('closes the context toolbar when it opens', async () => {
+    const editor = hook.editor();
+    editor.setContent('<p class="ctx-menu-me">Hello World</p>');
+
+    await Waiter.pTryUntil('Waiting for context toolbar to open', () => {
+      UiFinder.findIn(SugarBody.body(), '.tox-pop');
+    });
+
+    editor.setProgressState(true);
+    await pWaitForThrobber();
+
+    await Waiter.pTryUntil('context toolbar is closed', () => {
+      UiFinder.notExists(SugarBody.body(), '.tox-pop');
+    });
+  });
+
+  context('correctly re-opens (or does not re-open) the context toolbar when it closes', () => {
```

Comment:
I feel the `context` shouldn't specify what the tests are doing, but more the... context? It means we can add/change the unit tests without this becoming outdated. Also remember they'll all be printed out together in the test logs so we should avoid duplicating what it's doing

```suggestion
  context('context toolbar', () => {
```

Thoughts?

Reply:
Agreed

Reply:
Yeah? Would you also suggest moving the test above (another context toolbar thing) into this block then?